### DISCUSSION TITLE
SSH Tunnel could not be established: stderr: Host key verification failed

### DISCUSSION THREAD

**Discussion by gebvlad** [Vampire/teamcity-ssh-tunnel#3] [Q&A] [answered]
Hello, @Vampire 

I configured your plugin according to the scheme above. There is a problem:
`stderr: Host key verification failed` 
And build failed with `SSH Tunnel could not be established`

Screenshot
![image](https://user-images.githubusercontent.com/12844031/77830700-574e9b80-713b-11ea-996a-05d6756526c0.png)

I checked the port forwarding operation from the server on which TC works - everything works correctly. Command like this
```
ssh -fNL random_local_port:remote_addr:remote_port  proxy_user@proxy_server
```

Please, help me

**Comment by Vampire** [comment]
Hi, did you try this command on the server where TeamCity is running, or on the machine where the TeamCity agent is running? 
And did you execute the command as the user with which the TeamCity agent is running?

**Comment by gebvlad** [comment]
Yes, it's work fine:
```
[user@teamcity]$ ssh -fNL 10022:target_ip:10022  proxy_user@proxi_ip
[user@teamcity]$ ssh localhost -p 10022
The authenticity of host '[localhost]:10022 ([::1]:10022)' can't be established.
ECDSA key fingerprint is SHA256:############################.
ECDSA key fingerprint is MD5:##:##:##:##:##:##:##.
Are you sure you want to continue connecting (yes/no)? yes
Warning: Permanently added '[localhost]:10022' (ECDSA) to the list of known hosts.
Last login: Sat Mar 28 21:33:30 2020 from proxi_ip
[user@web]$
```

Also teamcity server and agents run at docker

**Comment by Vampire** [comment, marked as answer]
Sorry to ask again, but did you read that I wrote about the agent machine, not the server machine? And when I'm saying machine in your case it means from the agent docker image. I'm asking because the output makes me think you are running this from the server machine, not from the agent machine.

**Comment by Vampire** [comment]
Were you able to get it running properly now?

**Comment by Vampire** [comment]
As you don't answer, I'm assuming you either solved your issue or lost interest.
So I'm closing this issue.
If you still struggle with it, feel free to add a comment and I'll reopen the issue.
